src/executables/master_server_monitor.cpp

```diff
@@ -27,41 +27,38 @@
 
 #include "master_server_monitor.hpp"
 
+#include "error_manager.h"
+#include "system_parameter.h"
+
 std::unique_ptr<server_monitor> master_Server_monitor = nullptr;
 bool auto_Restart_server = false;
 
 server_monitor::server_monitor ()
 {
 
   m_server_entry_map = std::unordered_map<std::string, server_entry> ();
-  fprintf (stdout, "[SERVER_REVIVE_DEBUG] : server_entry_map is created. \n");
 
   {
     std::lock_guard<std::mutex> lock (m_server_monitor_mutex);
     m_thread_shutdown = false;
   }
 
   start_monitoring_thread ();
-
-  fflush (stdout);
+  _er_log_debug (ARG_FILE_LINE, "[Server Monitor] : Monitoring thread has been created.\n");
```

Comment:
```suggestion
  _er_log_debug (ARG_FILE_LINE, "[Server Monitor] Monitoring started.\n");
```

나머지도 콜론은 필요 없을 것 같습니다.

Reply:
수정하였습니다.